de.metas.swat/de.metas.swat.base/src/main/java/de/metas/adempiere/service/impl/CountryDAO.java

```diff
@@ -193,4 +196,24 @@ private void loadAllCountries(Properties ctx)
 				.listImmutable(I_C_Region.class);
 
 	}
+	
+	@Override
+	public List<I_C_Country_Sequence> retrieveCountrySequence(final I_C_Country country, final I_AD_Org org, final String language)
```

Comment:
shouldn't we consider caching all country sequences (maybe per country) and then this method shall only search in some in memory map?